gui/src/2D/atlasSourceImageManager.ts

```diff
@@ -0,0 +1,34 @@
+module BABYLON.GUI {
+    /**
+     * The class storage all atlas sources loaded
+     */
+    export class AtlasSourceImageManager {
+        private _data: any = {};
+        private _sourceImage: any = {};
+
+        constructor() {
+        }
+
+        /**
+         * Parse data from task loaded
+         * @param {BABYLON.JSONAssetTask} task
+         */

```

Comment:
This is where we should use the TextFileAssetTask directly
Also as an user I would expect to get a way to load it directly from an url